doc/BASICS.md

```diff
@@ -30,6 +30,7 @@ Optional options:
     -f, --file '.py'                 pr_file type to filter/trigger the test against: .py, .rb
     -d, --description 'DESCRIPTION'  Test decription
     -C, --check                      Check if there is any PR requiring a test, but do not run it.
+    -F, --force_test                 Force to run a test, even if is not marked to be re-triggered.
```

Comment:
I'm thinking in sake of minimalism if we could embed this option into the PR option, so we could avoid to add a new one. Imho having to much options can create a matrix of different variables and their combination that lead to bugs. If we have 8 flags we have more possibilities then 4. Number are take randomly here

Reply:
No we can't.
PR specify a concrete PR to review, but will not run the test if we don't have marked it for re-trigger.
But we can have special cases, like the one we have in my project, where a merged_enabled context must run even if we don't have a checkbox or a comment for it, as is kind of a hidden context.

Reply:
ok thx